frontend/src/service/chatService.ts

```diff
@@ -75,15 +76,19 @@ async function setGptModel(model: string): Promise<boolean> {
 }
 
 async function configureGptModel(
-	configId: MODEL_CONFIG,
+	configId: MODEL_CONFIG_ID,
 	value: number
-): Promise<boolean> {
+): Promise<ChatMessage | null> {
 	const response = await sendRequest(`${PATH}model/configure`, {
 		method: 'POST',
 		headers: { 'Content-Type': 'application/json' },
 		body: JSON.stringify({ configId, value }),
 	});
-	return response.status === 200;
+	const { resultingChatInfoMessage } =
+		(await response.json()) as ConfigureGptModelResponse;
```

Comment:
Yes this is the same problem as for defences: the response might not be JSON, plus we are swallowing update errors.

Reply:
cool, let's resolve that there and assume that it'll apply here as well